episodes/superspreading-simulate.Rmd

```diff
@@ -0,0 +1,851 @@
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- How can we simulate transmission chains based on infection characteristics?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Create a short term projection using a branching process with `{epichains}`.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: prereq
+
+## Prerequisites
+
+Learners should familiarise themselves with the following concept dependencies before working through this tutorial: 
+
+**Statistics**: Common probability distributions, including Poisson and negative binomial.
+
+**Epidemic theory**: The reproduction number, $R$.
+
+:::::::::::::::::::::::::::::::::
+
+## Introduction
+
+<!-- what we know -->
+
+Individual variation in transmission can affects both the potential for an epidemic to establish in a population and the ease of control ([Cori et al., 2017](https://royalsocietypublishing.org/doi/10.1098/rstb.2016.0371)). 
+
++ Greater variation reduces the overall probably of a single new case causing a large local outbreak, because most cases infect few others and individuals that generate a large number of secondary cases are relatively rare.
+
++ However, if a 'superspreading event' does occur and the outbreak gets established, this variation can make an outbreak harder to control using *mass interventions* (i.e. blanket interventions that implicitly assume everyone contributes equally to transmission), because some cases contribute disproportionality: a single uncontrolled case may generate a large number of secondary cases.
+
++ Conversely, variation in transmission may provide opportunities for *targeted interventions* if the individuals who contribute more to transmission (due to biological or behavioural factors), or the settings in which 'superspreading events' occur, share socio-demographic, environmental or geographical characteristics that can be defined.
+
+<!-- what we dont know -->
+
+How can we quantify the potential of a new infection to cause a large outbreak based on its reproduction number $R$ and the dispersion $k$ of its offspring distribution?
+
+```{r,echo=FALSE,warning=FALSE,message=FALSE,fig.cap="Observed number of cumulative cases from the Middle East respiratory syndrome (MERS) outbreak in South Korea, 2015, alongside with simulated transmission chains assuming an offspring distribution with $R=0.6$ and $k=0.02$."}
+library(epichains)
+library(epiparameter)
+library(tidyverse)
+library(outbreaks)
+
+mers_cumcases <- mers_korea_2015$linelist %>%
+  # incidence2 workflow
+  incidence2::incidence(date_index = "dt_onset") %>%
+  incidence2::complete_dates() %>%
+  # wrangling using {dplyr}
+  mutate(count_cumsum = cumsum(count)) %>%
+  rownames_to_column(var = "day") %>%
+  mutate(day = as.numeric(day))
+
+# offspring distribution
+mers_offspring <- c(mean = 0.60, dispersion = 0.02)
+
+# generation time
+serial_inverval <- epidist_db(
```

Comment:
```suggestion
serial_interval <- epidist_db(
```

Reply:
thanks! this propagated across the episode hehe

Reply:
fixed in https://github.com/epiverse-trace/tutorials-middle/commit/171028626419b00712a2f70699b319900431875c